cairo/ethereum_rlp/rlp.cairo

```diff
@@ -222,6 +222,48 @@ namespace ExtendedImpl {
         );
         return extended;
     }
+
+    func from_simple(simple: Simple) -> Extended {
```

Comment:
a bit unlucky here: 
rlp decode returns a `Simple`, but all the types in the trie.py/cairo (branch nodes, subnodes etc.) are `Extended`